pkgs/development/libraries/beignet/default.nix

```diff
@@ -107,5 +107,7 @@ stdenv.mkDerivation rec {
     license = licenses.lgpl21Plus;
     maintainers = with maintainers; [ artuuge zimbatm ];
     platforms = platforms.linux;
+    # Requires libdrm_intel
+    badPlatforms = [ "aarch64-linux" ];
```

Comment:
What is the semantics of badPlatforms?

Reply:
Right, I don't that described or even affecting any processing in nixpkgs.

Reply:
See b0482248fefbf3b6cdd9c92053cfb49778a3a3a8, platforms used are, simplified, `platforms - badplatforms`.

For beignet specifically, I would have preferred having a way to compose `linux × [i686, x86_64]`, but I don't think it's possible with how the platforms work currently.

Reply:
AFAIK this is the last remaining open "issue" about this PR. Anything required to change about the use of `badPlatforms`? Any better way to semantically say "x86-based platforms only" while still allowing to say "and linux" or "and either linux or darwin"?